features/orders.feature

```diff
@@ -0,0 +1,35 @@
+Feature: The order service back-end
+    As an Order Manager
+    I need a RESTful catalog service
+    So that I can keey track of all my orders
+
+Background:
+    Given the following orders:
+        | customer_id | status | 
+        | 5           | CONFIRMED    |
+        | 9           | SHIPPED      |
+        | 3           | SHIPPED      |
+        | 2           | DELIVERED    |
+        | 2           | CANCELLED    |
+        | 5           | IN_PROGRESS  |
+
+Scenario: The server is running
+    When I visit the "Home Page"
+    Then I should see "Order Demo RESTful Service" in the title
+    And I should not see "404 Not Found"
+
+Scenario: Create an Order
+    When I visit the "Home Page"
+    And I set the "Customer ID" to "5"
+    And I select "Confirmed" in the "Status" dropdown
```

Comment:
It won't matter since deserialization will just enum if there is a status value. There's no difference if it is the default, but sure I can change that. I can also add a test where only Customer ID is provided to test for default.

Reply:
It is fine to just change the value over here, no need to add a new test. Integration tests are just for testing the overall behavior. The rigorous testing for corner cases is already done by unit tests.